staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -7645,3 +7649,14 @@ const (
 	// the destination set to the node's IP and port or the pod's IP and port.
 	LoadBalancerIPModeProxy LoadBalancerIPMode = "Proxy"
 )
+
+// OCIVolumeSource represents a OCI volume resource.
+type OCIVolumeSource struct {
+	// Required: Image or artifact reference to be used.
+	Reference string `json:"reference" protobuf:"bytes,1,opt,name=reference"`
+
+	// Policy for pulling OCI objects.
```

Comment:
Describe how each of the policies works in practice in the godoc, so a user knows what Always means

Reply:
Added all possible values and docs as suggested.